app/Http/Controllers/FilesUploadController.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Http\Requests\FilesUploadRequest;
+use Intervention\Image\ImageManagerStatic as Image;
+use App\DTO\ImageResizeDTO;
+
+class FilesUploadController extends Controller
+{
+
+    public function upload(FilesUploadRequest $request)
+    {
+        $dto = new ImageResizeDTO();
+
+        $dto->setHeight(1300);
+        $dto->setWidth(1300);
+        $dto->setFolder('/upload/');
+        $dto->setCallback($request->CKEditorFuncNum);
+        $dto->setPath(public_path($dto->getFolder()));
+        $dto->setImage($request->file('upload'));
+        $dto->setFilename(md5(date("YmdHis") . rand(5, 50)) . '.' . $dto->getImage()->getClientOriginalExtension());
```

Comment:
What if there will be 2 same names for file? :)
Investigate an UUID generation and try it for filename.